region/crateria/east/West Ocean.json

```diff
@@ -1262,6 +1263,29 @@
         "It is necessary because there is a g-mode strat to get to 14."
       ]
     },
+    {
+      "link": [12, 14],
+      "name": "West Ocean - Enemy Stuck Moonfall Super Block Clip",
+      "notable": true,
+      "requires": [
+        "canEnemyStuckMoonfall",
+        "canTrickyUseFrozenEnemies",
+        "Morph",
+        {"or": [
+          "Bombs",
+          "SpringBall",
+          {"ammo": {"type": "PowerBomb", "count": 2}}
```

Comment:
Could save 1-2 PBs by starting from 6. If starting from 6, this strat would require 3 instead of 1.

Reply:
That would take freezing the bug repeatedly as you watch the tripper float left.
That is probably possible if you coordinate the freeze timer with the tripper being off camera so they end up in the right place at the same time 🤔

Reply:
Oh, nevermind, that sounds bad.